scripts/fix_nightly_feature_flags.py

```diff
@@ -0,0 +1,287 @@
+#!/bin/python3
+
+# This script checks that nightly feature flags are declared consistently so
+# we avoid nasty subtle bugs. See the individual checks below for details.
+
+import toml
+import re
+import difflib
+import sys
+
+
+class Crate:
+
+    def __init__(self, dir):
+        self.dir = dir
+        self.height = None
+        self.load_toml()
+
+    def load_toml(self):
+        self.toml = toml.load(self.dir + "/Cargo.toml")
+        self.name = self.toml["package"]["name"]
+        self.features = self.toml["features"] if 'features' in self.toml else {}
```

Comment:
(optional suggestion)
```suggestion
        self.features = self.toml.get("features", {})
```